features/order/managing_orders/browsing_orders.feature

```diff
@@ -0,0 +1,20 @@
+@managing_orders
+Feature: Browsing orders
+    In order to process orders
+    As an Administrator
+    I want to be able to browse new orders
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt"
+        And the store ships everywhere for free
+        And the store allows paying with "Cash on Delivery"
+        And there is a customer "john.doe@gmail.com" that placed an order "#00000022"
+        And the customer chose "Free" shipping method to "France" with "Cash on Delivery" payment
+        And the customer bought a single "PHP T-Shirt"
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Seeing a new order in the list
+        When I browse new orders
```

Comment:
`When I browse orders list` - on index page we have all orders not only the new ones :)